src/package/packageVersionCreate.ts

```diff
@@ -659,7 +659,15 @@ export class PackageVersionCreate {
     }
 
     const request = await this.createPackageVersionCreateRequestFromOptions();
-    const createResult = await this.connection.tooling.create('Package2VersionCreateRequest', request);
+    const options = {
+      multipartFileFields: {
+        VersionInfo: {
+          contentType: 'application/zip',
+          filename: 'package-version-info.zip',
+        },
+      },
+    };
+    const createResult = await this.connection.tooling.create('Package2VersionCreateRequest', request, options);
```

Comment:
This needs to happen with every package version create?  I.e., it's not optional?  Is there a performance impact?

Reply:
Technically if the payload is <25mb it does not need to use a mulitpart request. I considered branching the behavior but there should be no meaningful performance impact for using multipart requests in all cases so I opted for keeping it simple.